views/entry.ejs

```diff
@@ -8,5 +8,37 @@
 </head>
```

Comment:
You have to do all this in react native, so the extension should be js

Reply:
Okay